src/pages/index.js

```diff
@@ -88,19 +205,69 @@ profile
     profileEditFormValidation.resetValidation();
   });
 
+//////////////////////// Avatar Edit Form
+
+function handleAvatarFormSubmit(inputValues) {
+  api
+    .updateUserAvatar(inputValues)
+    .then((res) => {
+      if (res.ok) {
+        profileInfo.setUserAvatar(inputValues);
+      } else {
+        return Promise.reject(`Failed to update Avatar: ${res.status}`);
+      }
+    })
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**